examples/nft-create/README.md

```diff
@@ -107,14 +107,44 @@ Both are an identical transaction. The browser wallets tend to have better error
 
 ### Making localhost:3000 internet accessible
 
-When you scan the QR code it encodes the full URL of the checkout API, eg. `http://localhost:3000/api/checkout`. Without fiddling with networking on the phone, this can't be resolved by a mobile wallet.
+To enable your local development environment to be accessed from the internet—for example, to allow a mobile wallet to resolve and connect to your `localhost` server—you can use secure tunneling tools like Tunnelmole or ngrok.
 
-One easy way to handle this is to use [ngrok](https://ngrok.com). Once you sign up (free) and download their CLI you can run `ngrok http 3000`.
+#### Using Tunnelmole
 
-You'll see an output with a message like:
+[Tunnelmole](https://github.com/robbie-cahill/tunnelmole-client) is an open source tunneling tool that allows you to create a Public URL that forwards traffic to your local machine via a secure tunnel.
+
+To use Tunnelmole, install it first with the following command for Linux, Mac, and Windows Subsystem for Linux (WSL):
+
+```
+curl -O https://install.tunnelmole.com/z3469/install && sudo bash install
```

Comment:
Hmmm I'm not super comfortable asking devs (especially in this industry) to grab a shell script from the internet _and_ run it with root. Even if the open source project is safe, if that domain is compromised, everyone gets rekt and it could be very subtle.